PKHeX/PKHeX.Core.csproj

```diff
@@ -711,10 +712,8 @@
     <None Include="Resources\text\en\text_EncounterType_en.txt" />
     <None Include="Resources\text\de\text_EncounterType_de.txt" />
     <None Include="Resources\text\locale\regions3ds.txt" />
-    <None Include="Resources\text\locale\3dsregions.txt" />
```

Comment:
Is removing these files from the project intentional?  I don't see the files being deleted, so it looks accidental.